extension/src/popup/components/mnemonicPhrase/DisplayMnemonicPhrase/index.tsx

```diff
@@ -50,26 +45,6 @@ export const DisplayMnemonicPhrase = ({
           </div>
         </InfoBlock>
         <MnemonicDisplay mnemonicPhrase={mnemonicPhrase} />
-
-        {/* TODO - use Custom Button from SDS when added */}
-        <div className="DisplayMnemonicPhrase__display-buttons">
-          <button
-            className="DisplayMnemonicPhrase__download-button"
-            data-testid="download"
-            onClick={() => {
-              download({
-                filename: "freighterMnemonicPhrase.txt",
-                content: mnemonicPhrase,
-              });
-              emitMetric(METRIC_NAMES.accountCreatorMnemonicDownloadPhrase);
-            }}
-          >
-            <IconButton preset={IconButton.preset.download} />
-          </button>
-          <CopyText textToCopy={mnemonicPhrase} showTooltip>
-            <IconButton preset={IconButton.preset.copy} />
```

Comment:
maybe a question for bruno/charles too, but are we sure we want to remove the download/copy options? Since the vulnerability is about the browser saving the phrase unknowingly to users, where as download/copy is an active action by the user.

not saying we shouldn't remove, but just curious of your thoughts

Reply:
So for the copy method, Halborn specifically mentions preventing adding the mnemonic to the clipboard as it can leak into memory:

```
Prevent (and warn) users from directly copying/pasting the Mnemonic Phrase into the browser. The browser’s clipboard is also responsible for leaking data into memory. Therefore, it is important to warn the user not to fill the browser’s clipboard with the Mnemonic Phrase,```

They advocate for completing removing the ability to copy in any capacity.

Download is a little more of a gray area. I still haven't been able to determine how risky it is, but at this stage, I'm leaning towards being a bit more conservative. I think we'll probably learn a bit more about this vulnerability as Metamask/Phantom open-source their solutions. If at that point we determine any of these are not as risky as we thought, we can definitely bring them back

Reply:
gotchya, makes sense 👍